blockchain/vm/gas.go

```diff
@@ -38,21 +37,15 @@ const (
 
 // calcGas returns the actual gas cost of the call.
 //
-// The cost of gas was changed during the homestead price change HF. To allow for EIP150
-// to be implemented. The returned gas is gas - base * 63 / 64.
-func callGas(gasTable params.GasTable, availableGas, base uint64, callCost *big.Int) (uint64, error) {
-	if gasTable.CreateBySuicide > 0 {
-		availableGas = availableGas - base
-		gas := availableGas - availableGas/64
-		// If the bit length exceeds 64 bit we know that the newly calculated "gas" for EIP150
-		// is smaller than the requested amount. Therefor we return the new gas instead
-		// of returning an error.
-		if !callCost.IsUint64() || gas < callCost.Uint64() {
-			return gas, nil
-		}
-	}
-	if !callCost.IsUint64() {
-		return 0, errGasUintOverflow
```

Comment:
I erased this code because it's redundant code! In geth, this code is executed when `gasTable.CreateBySuicide > 0` is false. It means that the code is only executed when the node's hardfork version is lower than tangerineWhistle. However, klaytn forked the code after the tangerineWhistle, so we don't need this logic :)

Reply:
Thank you for the explanation. As we talked offline, `Constantinople` supports EIP150. Therfore only codes inside ` gasTable.CreateBySuicide > 0` is needed.

Reply:
@yoomee1313 
This erased code could be called when `gasTable.CreateBySuicide > 0` is true also.
After EIP 150, Is there no case if `!callCost.IsUint64()` is true?

Reply:
@KimKyungup 
I'm not sure if the answer to the question is correct. Feel free to ask a question if you have any doubts. 
`gasTable.createBySuicide > 0` is same meaning as `isEip150 == true`. You can see the changed geth code at [here](https://github.com/ethereum/go-ethereum/blob/22082f9e565665f7da045b9ff708e4698f383c77/core/vm/gas.go#L37-L53).
Since eip150 is default in klaytn, the condition is always true. Also, you can see there is only one GasTable in [here](https://github.com/klaytn/klaytn/blob/b2197d84603c67c0c99ce91ab821082d0050a608/params/gas_table.go#L44-L59).
In conclusion, the same code is executed twice every time(line 51 and line 55), so I removed the outside code of if loop.